util/summarize.py

```
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
```

Ruff (0.13.3)

Shebang is present but file is not executable

(EXE001)